### PR TITLE
Feature: Alerts & Buttons

### PR DESCRIPTION
**Pull request by athar05**
This branch includes alerts & button components for the component library.

### CODE REVIEW

**Comment by athar05** [comment]
Updates about development: 

Made changes to the grid structure as it broke. 

Identified and removed some redundant CSS classes. 

Fixed some minor bugs and glitches.

File: style.css [hunk 1]
```diff
@@ -234,4 +235,137 @@ body {
...
+  height: 2rem;
+  width: 15rem;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem 1rem;
+  border-radius: 0.5rem;
+}
+
+.alert.alert-success {
+  background-color: #d1e7dd;
```
Comment (raieegulhane) [inline style.css]: use variables for colors

**Review by raieegulhane — COMMENTED** [review]
Use variable for colors. Overall good to go.

**Review by ShoaibTamboli — COMMENTED** [review]
looks good to me. Just beautify/ Align the HTML and CSS files

**Pull request closed without merging**